postcss.config.js

```diff
@@ -1,6 +0,0 @@
-module.exports = {
-  plugins: [
-    require('postcss-inline-svg'),
-    require('cssnano'),
```

Comment:
**cssnano** reduces the size of the ``docsify-pagination.min.js`` file about 3kb. So I'll keep it.

Reply:
Sounds good. I didn't check to see the size different. I assumed (incorrectly) that it would negligible.